dask_expr/tests/test_groupby.py

```diff
@@ -34,9 +34,17 @@ def test_groupby_numeric(pdf, df, api, numeric_only):
     expect = getattr(pdf.groupby("x"), api)(numeric_only=numeric_only)
     assert_eq(agg, expect)
 
+    g = df.groupby("x")
+    agg = getattr(g, api)(numeric_only=numeric_only)["y"]
+
+    expect = getattr(pdf.groupby("x"), api)(numeric_only=numeric_only)["y"]
+    assert_eq(agg, expect)
+
 
 @pytest.mark.parametrize("func", ["count", "value_counts", "size"])
-def test_groupby_no_numeric_only(pdf, df, func):
+def test_groupby_no_numeric_only(pdf, func):
+    pdf = pdf.drop(columns="z")
```

Comment:
Curious about this?

Reply:
Never mind - I take it that this is just another variation on testing a projection after a groupby.